pkg/cloud/client.go

```diff
@@ -45,37 +46,37 @@ type client struct {
 	// csA *cloudstack.CloudStackClient
 }
 
+// cloud-config ini structure.
+type config struct {
```

Comment:
Turns out there's a perfectly natural way to use the `ini` lib that I wasn't using. Derp.

Reply:
Good to leverage this.
Looking down the road toward multi-endpoint, I envisioned having a section-per-endpoint, with a user-provided logical name for each section (by which that endpoint would be referred to).  Does the ini lib provide any means for enumerating the sections in an ini file if they're not known a-priori?